### PR TITLE
adds a tacti cool kit based off of neo

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
adds a syndicate tacti-cool bundle based off of neo from the matrix
currently its as rare as the dark lord one and contains
two stechkins 
sunglasses
2 ap clips
2 normal clips
1 incendiary
3 sepia extracts
a redpill injector (stimpak)
a syringe of plasma

thanks to mqiib for suggesting this
#### Changelog

:cl:  
rscadd: Added a new syndicate bundle
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
it failed on maps ???

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -194,7 +194,21 @@
 			new /obj/item/dnainjector/cryokinesis(src)
 			new /obj/item/gun/energy/temperature/security(src)
 			new /obj/item/melee/transforming/energy/sword/saber/blue(src) //see see it fits the theme bc its blue and ice is blue
-
+			
+			if("neo")
```
Comment (SomeguyManperson) [inline code/game/objects/items/storage/uplink_kits.dm]:
```suggestion
		if("neo")
```

Comment (SomeguyManperson) [reply code/game/objects/items/storage/uplink_kits.dm]: If statement was tabbed too many times that's what's causing it to not work

Comment (boodaliboo) [reply code/game/objects/items/storage/uplink_kits.dm]: thanks theos

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Sepia extracts won't activate them self, give a syringe with whatever the Chem is

**Review by ynot01 — CHANGES_REQUESTED** [review]
2 stechkin = 14tc
2 ap ammo = 4tc
2 ammo = 2tc
total = 20tc

if i wanted to waste my tc on ammo i wouldve just done it myself
give em a stimpack or something make the kit worth gambling for

**Comment by boodaliboo** [comment]
Ynot I’ll code in a red pill today and some more ammo

**Comment by boodaliboo** [comment]
> Ynot I’ll code in a red pill today and some more ammo

done also added plasma syringe for extract

**Pull request merged**